Lib/test/test_socket.py

```diff
@@ -6254,5 +6254,49 @@ def test_main():
     support.run_unittest(*tests)
     support.threading_cleanup(*thread_info)
 
+
+@requireAttrs(socket, "send_fds")
+@requireAttrs(socket, "recv_fds")
+@requireAttrs(socket, "AF_UNIX")
+class SendRecvFdsTests(unittest.TestCase):
+    @unittest.skipIf(sys.platform == 'darwin', 'test not working for MacOSX')
```

Comment:
This should be removed, I think @vstinner fix should work on MacOS as the error was the same. Fortunately, Azure pipelines have mac builders, so we do not need to run in the buildbots.